packages/connectors-lib/src/__tests__/document-client-decorator.spec.js

```diff
@@ -1,6 +1,6 @@
 import { createDocumentClient } from '../documentclient-decorator'
 import { DynamoDB } from '@aws-sdk/client-dynamodb'
-import { DynamoDBDocument } from '@aws-sdk/lib-dynamodb'
+import { DynamoDBDocument, QueryCommand, ScanCommand } from '@aws-sdk/lib-dynamodb'
```

Comment:
Was this change needed to make this change work or was it just some refactoring? (Not saying we shouldn't make it, I'm just curious!)